src/components/screens/login/networkSelector/networkSelector.js

```diff
@@ -44,23 +46,34 @@ const NetworkSelector = ({
   const [state, _setState] = useState(() => getInitialState(selectedAddress));
   const setState = newState => _setState(prevState => ({ ...prevState, ...newState }));
 
-  const onChangeInput = ({ target }) => {
-    const address = target.value;
-    setState({
-      address,
-      connected: false,
-      isValid: true,
-    });
+  const onChangeInput = (networkName, address) => {
+    if (networkName === networkKeys.customNode) {
+      setState({
+        address,
+        liskCustomNodeUrl: address,
+        connected: false,
+        isValid: true,
+      });
+    } else {
+      setState({
+        address,
+        connected: false,
+        isValid: true,
+      });
+    }
```

Comment:
That looks like some code repetition. We can define a new state before set state and then set it:
`let state = { address, connected: false, isValid: true };
if(...) { state.liskCustomNodeUrl = address }

setState(state); 
`